backend/substrapp/ledger_utils.py

```diff
@@ -338,7 +338,10 @@ def call_ledger(channel_name, call_type, fcn, *args, **kwargs):
         # add a log even if the function raises an exception
         te = time.time()
         elaps = (te - ts) * 1000
-        logger.info(f'smartcontract {call_type}:{fcn}; elaps={elaps:.2f}ms; error={error}')
+        if error is None:
+            logger.info(f"(smartcontract) {call_type}:{fcn} took {elaps:.2f} ms")
+        else:
+            logger.info(f"(smartcontract) {call_type}:{fcn} took {elaps:.2f} ms. Error: {error}")
```

Comment:
shouldn't use logger.error here?

Reply:
I'd argue that this log line is for informational purposes only, and that since the error is re-raised, it's the caller's responsibility to report the error as such. 

If we put `logger.error` here, then the logs will have **two** big red "ERROR" lines, suggesting that two different errors occurred. 

@inalgnu @Kelvin-M thoughts?

Reply:
I think it's better to differentiate logs that contain errors, especially if some exceptions are completely silent this would help to see that errors are happening even if it duplicates the information. 

However, I don't have a strong opinion on that so do whatever helps better-investigating errors.